lib/bigcommerce/connection.rb

```diff
@@ -34,24 +34,24 @@ def api_key=(api_key)
       @configuration[:api_key] = api_key
     end
 
-    def verify_peer=(verify)
+    def verify_ssl=(verify)
       @configuration[:verify_ssl] = verify
     end
 
     def ssl_ca_file=(path)
-      @configuration.ssl_ca_file = path
+      @configuration[:ssl_ca_file] = path
     end
 
-    def ssl_client_key=(path,passphrase=nil)
-      if passphrase.nil?
-        @configuration.ssl_client_key = OpenSSL::PKey::RSA.new(File.read(path))
+    def ssl_client_key=(options)
```

Comment:
We should keep the interface the same from the old version, the options hash is definitely nicer, but we dont want to break backward compatibility.